### PR TITLE
[Doc Improvement][Update on sideloading reference][3264909] 

### PR DESCRIPTION
**Pull request by v-ganr**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/tools-prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c6b7c0ca98590fe4131625ca4946bc0b8cd2f46/msteams-platform%2Ftoolkit%2Ftools-prerequisites.md) | 98 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/99739472-a7ac-4949-8b09-dccabc775844) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/tools-prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2d230422e8f11b012fd7d17cea227e2adf08b65f/msteams-platform%2Ftoolkit%2Ftools-prerequisites.md) | 97 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e9e49b2-da01-4f62-be53-ae99124df2a5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/tools-prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9e5365ea69da9a7abe16388740590de27074788c/msteams-platform%2Ftoolkit%2Ftools-prerequisites.md) | 97 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b3ec54d6-da40-42d1-982d-ce94ac9222f5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/tools-prerequisites.md [hunk 1]
```diff
@@ -13,14 +13,13 @@ Before you create your Teams app project, ensure that the prerequisites are in p
 
 * [Install required tools to build your Teams app](#install-required-tools-to-build-your-teams-app)
 * [Prepare Accounts to build your Teams app](#accounts-to-build-your-teams-app)
-* [Sideload permission](tools-prerequisites.md)
```
Comment (ChetanSharma-msft) [inline msteams-platform/toolkit/tools-prerequisites.md:16]: Why did we remove it. Please update it as per the provided link?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comment

File: msteams-platform/toolkit/tools-prerequisites.md (hunk 1 above)
Comment (v-ganr) [reply msteams-platform/toolkit/tools-prerequisites.md:16]: The suggested link is available in Verify sideloading permission. 

https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/tools-prerequisites#verify-sideloading-permission

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine, approving it.

**Review by v-deabira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/tools-prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4eb24a458161b5d734fe7001792ddf7c99d66ea6/msteams-platform%2Ftoolkit%2Ftools-prerequisites.md) | 97 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3167364e-ca2b-40a6-b5d6-e2db2b3b5325) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**